feature-libs/asm/root/services/asm-auth.service.ts

```diff
@@ -37,15 +38,17 @@ export class AsmAuthService extends AuthService {
     protected authStorageService: AsmAuthStorageService,
     protected authRedirectService: AuthRedirectService,
     protected globalMessageService: GlobalMessageService,
-    protected routingService: RoutingService
+    protected routingService: RoutingService,
+    protected authMultisiteIsolationService: AuthMultisiteIsolationService
```

Comment:
Let's make it an optional dep, not to introduce a breaking change.
```suggestion
    protected authMultisiteIsolationService?: AuthMultisiteIsolationService
```

btw. no need for `@Optional` decorator, because `AuthMultisiteIsolationService` is `providedIn: 'root'`

Reply:
Changed.